File: Data type/F_digits_summation.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    long long int n, m; // Use long long int to handle large values up to 10^18

    // Read input numbers
    cin >> n >> m;

    // Extract last digits using modulo operator (%)
    int nLastDigit = n % 10;
    int mLastDigit = m % 10;

    // Calculate and print the sum of last digits
    cout << (nLastDigit + mLastDigit) << endl;

    return 0;
}

File: Data type/F_digits_summation.py
# Read two space-separated numbers from input
n, m = map(int, input().split())

# Extract last digits using modulo operator (%)
n_last_digit = n % 10
m_last_digit = m % 10

# Calculate and print the sum of last digits
sum_of_last_digits = n_last_digit + m_last_digit
print(sum_of_last_digits)
